### ISSUE TITLE
AI flown Tie strikers with adaptive ailerons hardlocks the game if the aileron maneuver would fly them off the board

### ISSUE THREAD

**Issue by boredompwndu** [baranidlo/bench#15]
I suspect this has to do with the idea of a "default maneuver" that AI ships have (typically the shortest straight maneuver). If all maneuvers fly the ship off the table, they take the default maneuver, because it is unable to select an actual maneuver.

Because adaptive ailerons has no default option, and all other options fly the striker off the map, it just freezes.

**Comment by Sleet01** [comment]
It looks like more of an oversight in the IAUnits' out-of-activation maneuver handling code, which appears to make *no* choice and pass a negative value as an array index into the Adaptive Ailerons code.  Which is bad.

I've added a default left-bank-1-white if the IAUnit returns a -1 value, but only for Adaptive Ailerons.  Since I'm not sure exactly which situations cause the IAUnit code to return a -1 value, or how it's handled normally, I don't want to make a general change at this point.

This fix will be in the next PR.

**Comment by Sleet01** [comment]
This particular issue has been fixed, and some additional planning code fixes have also been implemented.  I believe we have one open bug on AI ships heading for the edge when they should preferably turn towards the board center (all other things being equal) but that is more of a superset of AI maneuvering than related to the TIE Striker.

tl;dr: closing.  Please open a new bug if further TIE Striker maneuver issues are encountered.
